assets/sass/components/header.scss

```diff
@@ -1,17 +1,72 @@
 .header {
+  position: relative;
   align-items: center;
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 1.2em;
-  padding: 0.6em 2%;
+  margin-bottom: 4em;
+  padding: 0.6em 2% 0.6em 2%;
+  border-bottom: 8px solid $color-primary;
+  transition-duration: 0.3s;
+  transition-property: all;
 
-  &__language-select {
+  @media only screen and (min-width: 712px) {
+    padding: 0.6em 110px 0.6em 2%;
+  }
+
+  &--sticky {
+    position: fixed;
+    background-color: white;
+    height: 70px;
+    width: 100%;
+    z-index: 100;
+    border-width: 2px;
+    top:0;
+
+    .logo {
+      position: absolute;
+      visibility: hidden;
+
+      @media only screen and (min-width: 712px) {
+        height: 11.9mm;
+        position: relative;
+        visibility: visible;
+      }
+    }
   }
 
   &__logo {
   }
 
   &__navigation {
     flex-grow: 1;
+
+    ul {
```

Comment:
Can you give this a classname also? BEM tries to avoid all nesting where possible to prevent specificity-wars. Something like `.header__navigation__links` could work.